### PR TITLE
WZ @ 5 TeV @ CMS @ LHC

### PR DESCRIPTION
**Pull request by cericeci**
PR content:
-elist argument to run over preselected events (by default off)
-Updated lepton ID (loose-tight) for WZ analysis
-Updated set of plots for WZ analysis
-Multithreaded WZ plotter for the people who hurry
-Some updated steps in WZ analysis to run in depth analysis

### CODE REVIEW

File: framework/analysis.py [hunk 1]
```diff
@@ -222,8 +222,8 @@ def printprocess(self, i):
     nevents = self.nRunEvents
     i0 = self.firstEvent
     if int(i) % int(self.nEventsPrintOut) == 0:
-      if   self.index == -1: print 'Processing... %i / %i (%1.2f'%(i,nevents, (float(i)-i0)/nevents*100) + '%)'
-      elif self.index == 0:  print 'Processing... %1.2f'%((float(i)-i0)/nevents*100) + ' %'
+      print 'Processing... %i / %i (%1.2f'%(i,nevents, (float(i)-i0)/nevents*100) + '%)'
```
Comment (GonzalezFJR) [inline framework/analysis.py]: Please

File: wzanalysis/wzanalysis.py [hunk 2]
```diff
@@ -101,12 +111,11 @@ def init(self):
     for i, dataName in dataset.items(): 
       if dataName == name: self.sampleDataset = i
 
-    # Jet and lep pT
+    # Jet global pT cuts
     self.JetPtCut  = 20
-    self.LepPtCut  = 8
-    self.Lep0PtCut = 20
```
Comment (GonzalezFJR) [inline wzanalysis/wzanalysis.py:107]: Why this?

**Review by GonzalezFJR — COMMENTED** [review]
For Carlos

File: wzanalysis/wzanalysis.py (hunk 2 above)
Comment (cericeci) [reply wzanalysis/wzanalysis.py:107]: They were not being used anywhere in the code

File: framework/analysis.py (hunk 1 above)
Comment (cericeci) [reply framework/analysis.py]: Restored to original less verbose syntax

**Pull request merged**